email_reminder/utils/reminder.py

```diff
@@ -3,11 +3,6 @@
 from frappe import _
 import re
 
-@frappe.whitelist()
-def fetch_reminder_doctypes():
```

Comment:
@zaqoutabed 
 why remove this !
how it get the doctype from reminder doctype ?